CatBlock/catblock
# Implement WebSocket blocking

WebSockets can be natively blocked by extensions starting in Chrome 58.

How-to test:
1. Add the following filter: `||pubsub-edge.twitch.tv/v1$websocket`
2. Go to `twitch.tv`
3. Open up the console and verify, that WebSocket requests were blocked

<img width="1269" alt="snimka obrazovky 2017-03-08 o 1 33 08" src="https://cloud.githubusercontent.com/assets/5178131/23684463/4dc70690-039f-11e7-8376-5111650cd323.png">
